packages/mui-base/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/base",
-  "version": "5.0.0-beta.40",
+  "version": "5.0.0-beta.41",
```

Comment:
Not sure if @mui/base is meant to be released or not.

Reply:
cc: @ZeeshanTamboli

Reply:
More context: https://github.com/mui/material-ui/issues/42177#issuecomment-2108399849

Reply:
> I think @mui/base shouldn't be released.

Maybe you're saying this because it's now legacy, but `TextField` with `multiline` depends on `TextareaAutosize` from `@mui/base`. Releasing this should fix #42177. For now, we can release from `master`. In the future, we make sure to avoid making fixes in `@mui/base`, even if Material UI depends on it.

Reply:
@mnajdova will be migrating code from the stale `@mui/base` to Material so we don't have to release it. I asked her to prioritize https://github.com/mui/material-ui/issues/42177. So the plan is skip releasing on this or future v5 releases, does that make sense to you @ZeeshanTamboli?

Reply:
PR for removing base dependency: https://github.com/mui/material-ui/pull/42907
@aarongarciah we can revert this bump as well

Reply:
Reverted